File: Listas de Exercícios (C)/C/Lista 1 - Casa/9.c
#include <stdio.h>

int main ()
{
    float P1, P2, A, M;
    
    printf("Média semestral do aluno\n\n");
    
    printf("--Valores decimais devem ser digitados com (.)!--\n");
    
    printf("Prova 1: "); scanf("%f", &P1);
    printf("Prova 2: "); scanf("%f", &P2);
    printf("Nota das atividades: "); scanf("%f", &A);
    
    
    M=(P1*4+P2*4+A*2)/10;
    
    printf("\nMédia semestral: %.1f\n", M);
}

File: 1th grade/Listas de Exercícios (Back-end [SPOLOGP])/Python/Lista 1 - Casa/9.py
print("Média semestral do aluno\n")

print("--Valores decimais devem ser digitados com (.)!--")

P1=float(input("Prova 1: "))
P2=float(input("Prova 2: "))
A=float(input("Nota das atividades: "))

M=(P1*4+P2*4+A*2)/10; M="{:.1f}".format(M)

print(f"\nMédia semestral: {M}\n")